contract.py

```diff
@@ -60,6 +62,7 @@ def configure(self, period: arc4.UInt64) -> None:
         self.enforce_step(UInt64(1)) # Fresh
         self.require_owner()
         assert period > 0, "period must be greater than 0" 
+        # JM: The Voi Foundation needs to verify that they are happy with this.
```

Comment:
Per conversation with Benedict we should allow a lockup period of 0 (>= 0). This will allow people to stake their tokens while they are vesting.